liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/PackageServerMojo.java

```diff
@@ -36,6 +36,18 @@
      */
     @Parameter(property = "packageFile")
     private File packageFile = null;
+    
+    /**
+     * Locate build directory
+     */
+    @Parameter(property = "project.build.directory")
+    private String projectBuildDir;
+    
+    /**
+     * Locate name of bundled project
+     */
+    @Parameter(property = "project.build.finalName")
+    private String projectBuildName;
```

Comment:
before Patrick had suggested that you use the following instead of injection:

private String projectBuildName = project.getBuild().getDirectory();